package.json

```diff
@@ -11,7 +11,10 @@
 		"url": "https://sindresorhus.com"
 	},
 	"type": "module",
-	"exports": "./index.js",
+	"exports": {
+		"default": "./index.js",
+		"browser": "./browser.js"
```

Comment:
The correct way to do it is like this: https://github.com/sindresorhus/ky-universal/blob/74c75592acd0af5cde6dac4e32f3e556a185a85c/package.json#L15-L16